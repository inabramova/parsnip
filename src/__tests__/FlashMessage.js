import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
import FlashMessage from '../components/FlashMessage';

Enzyme.configure({ adapter: new Adapter() });

describe('the TaskList component', () => {
  it('should render a status', () => {
    const errorText = 'some error just happened';
    const wrapper = shallow(<FlashMessage message={errorText} />);

    expect(wrapper.find('.flash-error').text()).toEqual(errorText);
  });

  it('should match the last snapshot with msg', () => {
    const wrapper = shallow(<FlashMessage message="oops" />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
