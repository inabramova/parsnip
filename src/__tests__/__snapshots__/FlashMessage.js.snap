// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the TaskList component should match the last snapshot with msg 1`] = `
<div
  className="flash-error"
>
  oops
</div>
`;
